venues/aclweb.org/NAACL/2022/post_suggested_decision_invitations.py

```diff
@@ -0,0 +1,124 @@
+import argparse
+from re import sub
+import openreview
+from tqdm import tqdm
+import csv
+import tracks
+
+"""
+OPTIONAL SCRIPT ARGUMENTS
+
+    baseurl -  the URL of the OpenReview server to connect to (live site: https://openreview.net)
+    username - the email address of the logging in user
+    password - the user's password
+
+"""
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+sac_name_dictionary = tracks.sac_name_dictionary
+
+with open('suggested_decision_pre_process.py') as g:
+    pre_content = g.read()
+    suggested_decision_super = openreview.Invitation(
+        id = "aclweb.org/NAACL/2022/Conference/-/Suggested_Decision",
+        readers = ["everyone"],
+        writers = ["aclweb.org/NAACL/2022/Conference"],
+        signatures = ["aclweb.org/NAACL/2022/Conference"],
+        reply = {
+            "readers":{
+                "values-copied": [
+                    "aclweb.org/NAACL/2022/Conference",
+                    "aclweb.org/NAACL/2022/Conference/Senior_Area_Chairs"
```

Comment:
missing a comma here at the end, but I guess it doesn't really matter. I think we don't usually set readers, writers and signatures in the super invitation, since we customize these fields for every paper when we submit paper invitations.